client/resource_group/controller/controller.go

```diff
@@ -269,7 +277,13 @@ func (c *ResourceGroupsController) shouldReportConsumption() bool {
 	if c.run.requestInProgress {
 		return false
 	}
-	timeSinceLastRequest := c.run.now.Sub(c.run.lastRequestTime)
+	maxDuration := func(a, b time.Duration) time.Duration {
+		if a > b {
+			return a
+		}
+		return b
+	}
+	timeSinceLastRequest := maxDuration(c.run.now.Sub(c.run.lastRequestTime), c.run.now.Sub(c.run.lastRequestTimeLowRU))
```

Comment:
This will cause resource groups to still request tokens during trickle time. How about lastRequestTime for each resource group?

Reply:
It's more changes, I think we can optimize this part in later. including:
- hibernate group
- each group maintain self report logic
- more metrics

Reply:
OK, hibernate group may be fixed by https://github.com/tikv/pd/pull/6098